CHANGELOG.md

```diff
@@ -22,11 +22,13 @@
 
 🔧 Fixes:
 
-- Pull Request Title goes here
+- Update \_summary-list.scss
```

Comment:
```suggestion
- Pull Request Title goes here

  Description goes here (optional)

  ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
 
- Minor CSS code change to prevent wrapping mid-word for most browsers
```

Reply:
It's not super obvious but we try and keep the template around for contributors in the future to copy and paste.

Also I think the sentence below 'update summary list' is a more descriptive title we can use.

Happy to do this for you when it gets merged, let me know.

Reply:
@nickcolley , sure.  Please do that for me, happy with the description change.  I'm still getting to grips with GitHub etc...